index.d.ts

```diff
@@ -1,8 +1,16 @@
-import React from 'react'
-import { GitHubButtonProps } from 'github-buttons'
+import React from 'react';
 
-interface ReactGitHubButtonProps extends GitHubButtonProps {
+export interface GitHubButtonProps extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
```

Comment:
Probably better to extends an intersection type? 
```
export interface ReactGitHubButtonProps extends GitHubButtonProps 
 & React.AnchorHTMLAttributes<HTMLAnchorElement> {
```

Reply:
Please @ntkme, check now.